src/SqlVirtualMachine/SqlVirtualMachine/SqlVirtualMachine/Cmdlet/NewAzureSqlVM.cs

```diff
@@ -75,11 +75,10 @@ public class NewAzureSqlVM : AzureSqlVMUpsertCmdletBase
         /// <summary>
         /// License type of new sql virtual machine
         /// </summary>
-        [Parameter(Mandatory = true,
```

Comment:
So now can the New-AzSqlVMConfig command have the licenseType set to false as well. The command is to create a local SQLVM object that can be then used to create a New SqlVM.
PS C:\> $config = New-AzSqlVMConfig -LicenseType "PAYG"
PS C:\> New-AzSqlVM -ResourceGroupName "ResourceGroup01" -Name "vm" -SqlVM $config